[18.0][MIG] hr_contract_multi_job: Migration to 18.0

xaviedoanhduy opened the pull request:
**note**

- remove the `index` attribute from an unstored data field
- I see a problem that a contract seems to be able to appear multiple times in the same job, do you think we should add a constraint to bind this on this occasion?

![image](https://github.com/user-attachments/assets/a9d7e984-4a4c-4000-a1ba-9ed46ceed66d)

miketelahun committed:
Employee and Contract Management, Attendance and Scheduling, Payroll Management modules

miketelahun committed:
Employee and Contract Management, Attendance and Scheduling, Payroll Management modules

Commit:
PEP8

dreispt committed:
Initialize 8.0 branch

dreispt committed:
Initialize 8.0 branch

feketemihai committed:
Port hr_contract_reference to V8 and new api.

feketemihai committed:
Add readme.

feketemihai committed:
Update readme.

feketemihai committed:
Update code as commented, update readme.

feketemihai committed:
Fix travis errors.

feketemihai committed:
Update readme.

feketemihai committed:
Update authors, license definition.

feketemihai committed:
Fix typo.

yvaucher committed:
Add missing default oca icons

saltonmassally committed:
hr_employee_id: Port to V8

Initial Attempt of a port to V8
Some improvements made are as follows:
* Ability so choose ID generation method
* Sequence vs Random

hr_employee_id: code linting and copyright removals

apply suggeste changes
* Use identification_id provided by hr addon rather than employee_no
* use new api in configuration
* pep8 changes

Update res_company.py

make fields assignable from res_company

add tests

Remove duplicate license  key

import openerp.exception.Warning as UserWarning

[FIX] import Warning as UserWarning

oca-transbot committed:
OCA Transbot updated translations from Transifex

sbidoul committed:
prefix versions with 8.0

sbidoul committed:
[MOV] move addons out of __unported__ (they remain not installable)

andhit-r committed:
[MIG] hr_employee_id: Migration to 8.0

* Change .py header according OCA simple header
* Add parameter name on field declaration
* Greenify travis
* Modify README.rst. Removing tabs from xml
* Remove space from xml files
* Make module installable

pedrobaeza committed:
Make modules uninstallable

pedrobaeza committed:
Rename manifest files

astirpe committed:
[MIG] 10.0 Porting hr_employee_id (#272)

leemannd committed:
hr_contract_reference: Migrated to 10.0

simahawk committed:
[fix] `hr_employee_id` use deprecated `sequence.get_id()` (#358)

simahawk committed:
[imp] hr_employee_id: ease override w/ custom ID

oca-travis committed:
Update hr_contract_reference.pot

Trivedi-Vacha-SerpentCS committed:
[MIG][11.0]hr_contract_multi_jobs:Migrated from 10.0 to 11.0.

Trivedi-Vacha-SerpentCS committed:
hr_contract_reference:Migration to 11.0.

Trivedi-Vacha-SerpentCS committed:
[MIG][11.0]hr_employee_id:Migration to 11.0.

luistorresm committed:
[MIG] hr_contract_multi_job: Migration from v11 to v12

luistorresm committed:
[IMP] hr_contract_multi_job: Fix traceback by singleton error.

If you try set two jobs like main position, first come in the computed
and after in the constraint.

To avoid the traceback, is assigned only the main job position when is
found a record, and when you try save the record is generated the
Warning in the traceback.

oca-travis committed:
Update hr_contract_reference.pot

alexey-pelykh committed:
[MIG] hr_employee_id: Migration to 12.0

alexey-pelykh committed:
[12.0][ADD] hr_employee_service: employee service information & duration

alexey-pelykh committed:
[12.0][ADD] hr_employee_service_contract: calculate service length based on contracts

[UPD] Update hr_employee_service_contract.pot

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (1 of 1 strings)

Translation: hr-11.0/hr-11.0-hr_contract_reference
Translate-URL: https://translation.odoo-community.org/projects/hr-11-0/hr-11-0-hr_contract_reference/de/

alexey-pelykh committed:
[FIX] hr_employee_service_contract: remove duplicate view

[UPD] Update hr_employee_service_contract.pot

[ADD] icon.png

osoul-dev committed:
Translated using Weblate (Arabic)

Currently translated at 53.3% (8 of 15 strings)

Translation: hr-12.0/hr-12.0-hr_employee_id
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_id/ar/

Nikul-Chaudhary committed:
[MIG] hr_contract_reference v11 to v12

mvrodriguez committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (16 of 16 strings)

Translation: hr-12.0/hr-12.0-hr_employee_id
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_id/es/

OCA-git-bot committed:
[ADD] icon.png

Commit:
Added translation using Weblate (French)

Commit:
Translated using Weblate (French)

Currently translated at 93.8% (15 of 16 strings)

Translation: hr-12.0/hr-12.0-hr_employee_service
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_service/fr/

Commit:
Added translation using Weblate (French)

Commit:
Translated using Weblate (French)

Currently translated at 88.9% (8 of 9 strings)

Translation: hr-12.0/hr-12.0-hr_employee_service_contract
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_service_contract/fr/

Commit:
[12.0][ADD] Migration script from hr_employee_seniority

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (9 of 9 strings)

Translation: hr-12.0/hr-12.0-hr_employee_service_contract
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_service_contract/es/

[UPD] README.rst

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (16 of 16 strings)

Translation: hr-12.0/hr-12.0-hr_employee_service
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_service/es/

marylla committed:
Translated using Weblate (German)

Currently translated at 81.2% (13 of 16 strings)

Translation: hr-12.0/hr-12.0-hr_employee_id
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_id/de/

alexey-pelykh committed:
[IMP] hr_employee_service: support _get_date_start_work

mvasquez13 committed:
[FIX]  hr_employee_service: Don't try to access field service_duration for getting label

It will fail if no permissions

Commit:
[12.0][FIX] hr_employee_service: Test 3

flotho committed:
Fix bad syntax in query

the query is not correct because of semicolon

marylla committed:
Added translation using Weblate (German)

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (9 of 9 strings)

Translation: hr-12.0/hr-12.0-hr_employee_service_contract
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_service_contract/de/

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 81.2% (13 of 16 strings)

Translation: hr-12.0/hr-12.0-hr_employee_id
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_id/hr/

MiquelRForgeFlow committed:
[FIX] Avoid not intended warnings

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (16 of 16 strings)

Translation: hr-12.0/hr-12.0-hr_employee_id
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_id/pt_BR/

marylla committed:
Added translation using Weblate (German)

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (16 of 16 strings)

Translation: hr-12.0/hr-12.0-hr_employee_service
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_service/de/

Saran440 committed:
[12.0][MIG] hr_employee_language

Saran440 committed:
[IMP] Add boolean can_listen and change menu position

Saran440 committed:
[IMP] hr_employee_language: black, isort

Saran440 committed:
[13.0][MIG] hr_employee_language

Saran440 committed:
[IMP] hr_employee_service: black, isort

Saran440 committed:
[13.0][MIG] hr_employee_service

kranokporn committed:
[IMP] hr_employee_service_contract: black, isort, prettier

kranokporn committed:
[MIG] hr_employee_service_contract: Migration to 13.0

ps-tubtim committed:
[IMP] hr_contract_multi_job: black, isort, prettier

ps-tubtim committed:
[13.0][MIG] hr_contract_multi_job

oca-travis committed:
[UPD] Update hr_contract_multi_job.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-13.0/hr-13.0-hr_contract_multi_job
Translate-URL: https://translation.odoo-community.org/projects/hr-13-0/hr-13-0-hr_contract_multi_job/

Commit:
[FIX] hr_employee_service_contract: _compute when date_end changed

oca-travis committed:
[UPD] Update hr_contract_reference.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_contract_reference
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_contract_reference/

Commit:
[IMP] hr_contract_reference: black, isort, prettier

Commit:
[13.0][MIG] hr_contract_reference

Saran440 committed:
[FIX] store compute field

[UPD] Update hr_employee_service_contract.pot

[UPD] README.rst

oca-travis committed:
[UPD] Update hr_employee_language.pot

alexey-pelykh committed:
[UPD] Brainbean Apps => CorporateHub

[UPD] README.rst

aphon61bank committed:
[14.0][MIG] hr_employee_service

aphon61bank committed:
[14.0][MIG] hr_contract_multi_job

oca-travis committed:
[IMP] hr_contract_reference: black, isort, prettier

Squashed commits:
[ba6c760d0] [UPD] README.rst
[6e4781cb3] [UPD] Update hr_contract_reference.pot

Commit:
[MIG] hr_contract_reference: Migration to 14.0

lk-eska committed:
use built-in departure date as termination date

change quotes

fix

fix

iii

fix

iii

britoederr committed:
Added translation using Weblate (Portuguese (Brazil))

britoederr committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (19 of 19 strings)

Translation: hr-14.0/hr-14.0-hr_employee_service
Translate-URL: https://translation.odoo-community.org/projects/hr-14-0/hr-14-0-hr_employee_service/pt_BR/

oca-travis committed:
[UPD] Update hr_contract_reference.pot

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update hr_contract_multi_job.pot

OCA-git-bot committed:
[UPD] README.rst

Commit:
Added translation using Weblate (Catalan)

Commit:
Translated using Weblate (Catalan)

Currently translated at 89.4% (17 of 19 strings)

Translation: hr-14.0/hr-14.0-hr_employee_service
Translate-URL: https://translation.odoo-community.org/projects/hr-14-0/hr-14-0-hr_employee_service/ca/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (2 of 2 strings)

Translation: hr-14.0/hr-14.0-hr_contract_reference
Translate-URL: https://translation.odoo-community.org/projects/hr-14-0/hr-14-0-hr_contract_reference/de/

mtelahun committed:
[IMP] hr_employee_id: black, isort, prettier

mtelahun committed:
Fix hr_employee_id: domain on non-relational field makes no sense

mtelahun committed:
hr_employee_id: Add additional tests

o test default values even when company fields are false
o test multi-company correctness

obayit committed:
[FIX] hr_employee_service: use sudo

fixes #1015
Unit tests for `hr_holidays` will not pass without using sudo here.

etobella committed:
[FIX] hr_employee_service_contract: Set fields as not prefetched

hr_employee_service_contract 13.0.1.0.1

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: hr-14.0/hr-14.0-hr_contract_reference
Translate-URL: https://translation.odoo-community.org/projects/hr-14-0/hr-14-0-hr_contract_reference/pt_BR/

sbidoul committed:
pre-commit: fix manifest url

Noel000 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (19 of 19 strings)

Translation: hr-14.0/hr-14.0-hr_employee_service
Translate-URL: https://translation.odoo-community.org/projects/hr-14-0/hr-14-0-hr_employee_service/ca/

pedrobaeza committed:
[FIX] hr_employee_id: Avoid W7940 (dangerous-view-replace-wo-priority)

olgamarcocb committed:
[MIG] hr_employee_service: Migration to 15.0

albariera committed:
[MIG] hr_employee_service_contract: Migration to 14.0

[UPD] Update hr_employee_service_contract.pot

[UPD] README.rst

Commit:
[UPD] Update hr_contract_multi_job.pot

Commit:
[UPD] Update hr_contract_reference.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-14.0/hr-14.0-hr_contract_multi_job
Translate-URL: https://translation.odoo-community.org/projects/hr-14-0/hr-14-0-hr_contract_multi_job/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-14.0/hr-14.0-hr_contract_reference
Translate-URL: https://translation.odoo-community.org/projects/hr-14-0/hr-14-0-hr_contract_reference/

Commit:
[UPD] Update hr_employee_service.pot

OCA-git-bot committed:
[UPD] README.rst

olgamarcocb committed:
[MIG] hr_employee_service_contract: Migration to 15.0

Commit:
[UPD] Update hr_employee_service_contract.pot

OCA-git-bot committed:
[UPD] README.rst

sanchonuria committed:
[MIG] hr_contract_reference: Migration to 15.0

Commit:
[UPD] Update hr_contract_reference.pot

OCA-git-bot committed:
[UPD] README.rst

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translation: hr-15.0/hr-15.0-hr_contract_reference
Translate-URL: https://translation.odoo-community.org/projects/hr-15-0/hr-15-0-hr_contract_reference/pt_BR/

mtelahun committed:
[MIG] hr_employee_id: Migration to 15.0

Commit:
[UPD] Update hr_employee_id.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-15.0/hr-15.0-hr_employee_id
Translate-URL: https://translation.odoo-community.org/projects/hr-15-0/hr-15-0-hr_employee_id/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 47.3% (9 of 19 strings)

Translation: hr-14.0/hr-14.0-hr_contract_multi_job
Translate-URL: https://translation.odoo-community.org/projects/hr-14-0/hr-14-0-hr_contract_multi_job/it/

yaseentai committed:
[MIG] hr_contract_multi_job: Migration to 15.0

yaseentai committed:
[Fix} pre-commit run

Commit:
[UPD] Update hr_contract_reference.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-15.0/hr-15.0-hr_contract_reference
Translate-URL: https://translation.odoo-community.org/projects/hr-15-0/hr-15-0-hr_contract_reference/

yaseentai committed:
[15.0][MIG] hr_contract_multi_job: Migration to 15.0

Commit:
[MIG] hr_contract_reference: Migration to 16.0

nimarosa committed:
[14.0] [IMP] hr_employee_service: Add helper function

nimarosa committed:
[14.0] [IMP] hr_employee_service: Add tests of helper function

[14.0] [IMP] hr_employee_service: Add tests of helper function

Commit:
[UPD] Update hr_contract_multi_job.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-15.0/hr-15.0-hr_contract_multi_job
Translate-URL: https://translation.odoo-community.org/projects/hr-15-0/hr-15-0-hr_contract_multi_job/

Commit:
[MIG] hr_employee_service_contract: Migration to 16.0

Commit:
[MIG] hr_employee_service: Migration to 16.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (19 of 19 strings)

Translation: hr-15.0/hr-15.0-hr_contract_multi_job
Translate-URL: https://translation.odoo-community.org/projects/hr-15-0/hr-15-0-hr_contract_multi_job/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (16 of 16 strings)

Translation: hr-15.0/hr-15.0-hr_employee_id
Translate-URL: https://translation.odoo-community.org/projects/hr-15-0/hr-15-0-hr_employee_id/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (16 of 16 strings)

Translation: hr-15.0/hr-15.0-hr_employee_id
Translate-URL: https://translation.odoo-community.org/projects/hr-15-0/hr-15-0-hr_employee_id/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (107 of 107 strings)

Translation: hr-13.0/hr-13.0-hr_employee_language
Translate-URL: https://translation.odoo-community.org/projects/hr-13-0/hr-13-0-hr_employee_language/it/

JoanSForgeFlow committed:
[ADD] hr_employee_birthday_mail: birthday message and coworkers alert

Commit:
[UPD] Update hr_employee_birthday_mail.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (25 of 25 strings)

Translation: hr-16.0/hr-16.0-hr_employee_birthday_mail
Translate-URL: https://translation.odoo-community.org/projects/hr-16-0/hr-16-0-hr_employee_birthday_mail/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (107 of 107 strings)

Translation: hr-13.0/hr-13.0-hr_employee_language
Translate-URL: https://translation.odoo-community.org/projects/hr-13-0/hr-13-0-hr_employee_language/it/

JoanSForgeFlow committed:
[FIX] hr_employee_birthay_mail: res_users configurable

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (25 of 25 strings)

Translation: hr-16.0/hr-16.0-hr_employee_birthday_mail
Translate-URL: https://translation.odoo-community.org/projects/hr-16-0/hr-16-0-hr_employee_birthday_mail/es/

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (19 of 19 strings)

Translation: hr-15.0/hr-15.0-hr_contract_multi_job
Translate-URL: https://translation.odoo-community.org/projects/hr-15-0/hr-15-0-hr_contract_multi_job/es/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-15.0/hr-15.0-hr_contract_multi_job
Translate-URL: https://translation.odoo-community.org/projects/hr-15-0/hr-15-0-hr_contract_multi_job/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-16.0/hr-16.0-hr_employee_birthday_mail
Translate-URL: https://translation.odoo-community.org/projects/hr-16-0/hr-16-0-hr_employee_birthday_mail/

AlexPForgeFlow committed:
[MIG] hr_contract_multi_job: Migration to 16.0

remi-filament committed:
[MIG] hr_employee_service: fix tests and remove obsolete function

gva-acsone committed:
Added translation using Weblate (French)

gva-acsone committed:
Added translation using Weblate (French (France) (fr_FR))

gva-acsone committed:
Translated using Weblate (French)

Currently translated at 4.0% (1 of 25 strings)

Translation: hr-16.0/hr-16.0-hr_employee_birthday_mail
Translate-URL: https://translation.odoo-community.org/projects/hr-16-0/hr-16-0-hr_employee_birthday_mail/fr/

Commit:
[UPD] Update hr_employee_service.pot

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (16 of 16 strings)

Translation: hr-16.0/hr-16.0-hr_employee_service
Translate-URL: https://translation.odoo-community.org/projects/hr-16-0/hr-16-0-hr_employee_service/it/

maisim committed:
[MIG] hr_employee_id: Migration to 16.0

Commit:
[UPD] Update hr_contract_reference.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-16.0/hr-16.0-hr_contract_reference
Translate-URL: https://translation.odoo-community.org/projects/hr-16-0/hr-16-0-hr_contract_reference/

maisim committed:
[IMP] hr_employee_id: Add support for batch creation

- Updated the create method in models/hr_employee.py to not break batch mode create

maisim committed:
[IMP] hr_employee_id: Improve tests coverage

- Add test : test_generate_identification_id_exception

Commit:
[UPD] Update hr_employee_id.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-16.0/hr-16.0-hr_employee_id
Translate-URL: https://translation.odoo-community.org/projects/hr-16-0/hr-16-0-hr_employee_id/

Commit:
[UPD] Update hr_employee_service_contract.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update hr_employee_birthday_mail.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-16.0/hr-16.0-hr_employee_birthday_mail
Translate-URL: https://translation.odoo-community.org/projects/hr-16-0/hr-16-0-hr_employee_birthday_mail/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (26 of 26 strings)

Translation: hr-16.0/hr-16.0-hr_employee_birthday_mail
Translate-URL: https://translation.odoo-community.org/projects/hr-16-0/hr-16-0-hr_employee_birthday_mail/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (9 of 9 strings)

Translation: hr-16.0/hr-16.0-hr_employee_service_contract
Translate-URL: https://translation.odoo-community.org/projects/hr-16-0/hr-16-0-hr_employee_service_contract/it/

ArnauCForgeFlow committed:
[IMP] hr_employee_birthday_mail: pre-commit auto fixes

ArnauCForgeFlow committed:
[MIG] hr_employee_birthday_mail: Migration to 17.0

etobella committed:
[IMP] hr_employee_service: Remove mock dependancy

ArnauCForgeFlow committed:
[IMP] hr_employee_birthday_mail: pre-commit auto fixes

ArnauCForgeFlow committed:
[MIG] hr_employee_birthday_mail: Migration to 18.0

Commit:
[UPD] Update hr_contract_multi_job.pot

OCA-git-bot committed:
[BOT] post-merge updates

luisDIXMIT committed:
[IMP] hr_employee_service: pre-commit auto fixes

luisDIXMIT committed:
[MIG] hr_employee_service: Migration to 18.0

luisDIXMIT committed:
[IMP] hr_employee_service_contract: pre-commit auto fixes

luisDIXMIT committed:
[MIG] hr_employee_contract: Migration to 18.0

luisDIXMIT committed:
[IMP] hr_contract_reference: pre-commit auto fixes

luisDIXMIT committed:
[MIG] hr_contract_reference: Migration to 18.0

luisDIXMIT committed:
[IMP] hr_employee_id: pre-commit auto fixes

luisDIXMIT committed:
[MIG] hr_employee_id: Migration to 18.0

xaviedoanhduy committed:
[IMP] hr_contract_multi_job: pre-commit auto fixes

xaviedoanhduy committed:
[MIG] hr_contract_multi_job: Migration to 18.0

nguyenminhchien commented on hr_contract_multi_job/views/hr_contract_view.xml:
```diff
@@ -0,0 +1,24 @@
+        <field name="inherit_id" ref="hr_contract.hr_contract_view_form" />
+        <field name="arch" type="xml">
+            <field name="job_id" position="attributes">
+                <attribute name="readonly">1</attribute>
```
There are 2 field tags of `job_id`. So, this does not work.
![image](https://github.com/user-attachments/assets/56a8245e-3fda-4991-abe4-aa569685bfe0)

xaviedoanhduy replied:
Actually, there are 2 `job_id` fields in previous versions, I think the author's purpose is just to assign `readonly` attribute to the first job_id field, which is the field that requires the user not to be in the contract manager group. On the contrary, the manager can edit `job_id` without following the current module.

xaviedoanhduy replied:
I tested it by changing its label (Job Title 1). And if the admin wants to change this value manually, it still works.

![image](https://github.com/user-attachments/assets/434a173b-10ee-4fe5-b50e-930e52f65242)

nguyenminhchien commented:
> I see a problem that a contract seems to be able to appear multiple times in the same job, do you think we should add a constraint to bind this on this occasion?

I think, we should keep it as it is.

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #1466 into 18.0

Signed-off-by dreispt

Commit:
[UPD] Update hr_employee_id.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
[IMP] hr_employee_language: black, isort, prettier

mymage committed:
[MIG] hr_employee_language: Migration to 16.0

mymage committed:
[IMP] hr_employee_language: pre-commit auto fixes

mymage committed:
[MIG] hr_employee_language: Migration to 17.0

mymage committed:
[IMP] hr_employee_language: pre-commit auto fixes

mymage committed:
[MIG] hr_employee_language: Migration to 18.0

victoralmau committed:
[FIX] hr_employee_medical_examination: Use _for_xml_id() to avoid access errors

TT56829

OCA-git-bot committed:
Merge PR #1482 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update hr_employee_medical_examination.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-18.0/hr-18.0-hr_employee_medical_examination
Translate-URL: https://translation.odoo-community.org/projects/hr-18-0/hr-18-0-hr_employee_medical_examination/

OCA-git-bot committed:
Merge PR #1478 into 18.0

Signed-off-by rvalyi

Commit:
[UPD] Update hr_employee_language.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1444 into 18.0

Signed-off-by etobella

Commit:
[UPD] Update hr_employee_birthday_mail.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1463 into 18.0

Signed-off-by etobella

Commit:
[UPD] Update hr_employee_service.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1465 into 18.0

Signed-off-by etobella

Commit:
[UPD] Update hr_contract_reference.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1464 into 18.0

Signed-off-by etobella

Commit:
[UPD] Update hr_employee_service_contract.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: hr-18.0/hr-18.0-hr_contract_reference
Translate-URL: https://translation.odoo-community.org/projects/hr-18-0/hr-18-0-hr_contract_reference/it/

victoralmau committed:
[IMP] hr_employee_medical_examination: Add Activities filter + kanban view

TT57402

victoralmau committed:
[IMP] hr_employee_medical_examination: Edit the model description to remove "HR"

Related to https://github.com/OCA/hr/pull/1490#pullrequestreview-3068562926

TT57402

ChristophAbenthungCibex committed:
[FIX]hr_employee_calendar_planning: Fix _check_company_id

The check does not consider the calendar itself. It only checks if any calendar has inconsistency with companies.

OCA-git-bot committed:
Merge PR #1494 into 18.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1496 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update hr_employee_medical_examination.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-18.0/hr-18.0-hr_employee_medical_examination
Translate-URL: https://translation.odoo-community.org/projects/hr-18-0/hr-18-0-hr_employee_medical_examination/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (77 of 77 strings)

Translation: hr-18.0/hr-18.0-hr_employee_medical_examination
Translate-URL: https://translation.odoo-community.org/projects/hr-18-0/hr-18-0-hr_employee_medical_examination/it/

victoralmau committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (77 of 77 strings)

Translation: hr-18.0/hr-18.0-hr_employee_medical_examination
Translate-URL: https://translation.odoo-community.org/projects/hr-18-0/hr-18-0-hr_employee_medical_examination/es/
